README.md

```diff
@@ -61,18 +54,25 @@ The list of ports that are used for unit tests and sample code:
 | 8073 | reseller H2 Console port |
```

Comment:
Can we remove all H2 console port ? As we use MariaDB by default. MariaDB uses port 3306 for comms.